actions/crib-deploy-environment/action.yml

```diff
@@ -211,6 +218,49 @@ runs:
         fi
 
         nix develop -c devspace deploy --skip-build -o=${{ inputs.product-image-tag }} "${profile_args_array[@]}"
+
+    - name: Get List of the Ingress Services
+      shell: bash
+      id: get-ingress-services
+      run: |
+        BASE_DOMAIN="${{ inputs.ingress-base-domain }}"
+        NAMESPACE="${{ steps.generate-ns-name.outputs.devspace-namespace }}"
+
+        # Fetch all Ingress hostnames from the specified namespace
+        ingresses=$(kubectl get ingress -n "$NAMESPACE" -o jsonpath='{.items[*].spec.rules[*].host}' 2>/dev/null)
+
+        # Check if we received any hostnames
+        if [[ -z "$ingresses" ]]; then
+          echo "No Ingress hostnames found in namespace $NAMESPACE."
+          exit 1
+        fi
+
+        # Prepare the output string for environment variable
+        OUTPUT_STRING=""
+
+        # Loop through each ingress entry
+        for host in $ingresses; do
+          # Remove the domain from the host
+          base_name=${host%.$BASE_DOMAIN}
+
+          # Extract the prefix up to the first dot
+          prefix=${base_name%%.*}
+
+          # Remove the namespace from the prefix
+          key=${prefix#$NAMESPACE-}
+
+          # Replace hyphens with underscores and convert to uppercase
+          key=$(echo "$key" | tr '-' '_' | tr '[:lower:]' '[:upper:]')
+
+          # Append to output string in the format KEY=base_name
+          OUTPUT_STRING+="$key=$base_name"$'\n'
```

Comment:
How can someone easily find the list of available keys without running this step in the action?

Reply:
I'll document this but it's easy to figure out because the naming scheme is <namespace-name>-<service-name>.<$main-domain>